assets/js/analysis/index.controller.js

```diff
@@ -18,6 +18,10 @@
     $scope.findCorpus = function (id) {
       return _.find($scope.corpora, {'_id': id});
     }
-
+    $scope.getEtaTime = function(analysis) {
+      var d = new Date(analysis.time_created); 
+      d.setSeconds(d.getSeconds() + analysis.eta);
+      return d.toLocaleTimeString()
```

Comment:
We might want to do `return d.toLocaleString()` here.  Analyses probably won't take multiple days, but I could see people getting weirded out by looking at old analyses that they did days ago and just seeing "ETA: 6:28:48 PM" in the metadata.  

That being said, it might be a better idea to hide ETA once the analysis is complete.